iOS/Controllers/HomeworkList/HomeworkSubmitViewController.swift

```diff
@@ -167,7 +167,13 @@ final class HomeworkSubmitViewController: UIViewController {
             self.navigationItem.setLeftBarButton(nil, animated: true)
         }
 
-        self.commentField.text = self.writableSubmission.comment ?? placeholder
+        if let comment = self.writableSubmission.comment,
```

Comment:
Should we also trim the comment for leading and trailing whitespaces to avoid strings only containing whitespace?

Reply:
I think it's a good idea, will add.

Reply:
To be defensive, we should also trim here. Since you are checking here for an empty string and modification to submissions can also occur also on other systems.